rs/tests/common.bzl

```diff
@@ -11,7 +12,13 @@ GUESTOS_RUNTIME_DEPS = [
     "//ic-os/components:hostos-scripts/build-bootstrap-config-image.sh",
 ]
 
-MAINNET_REVISION_RUNTIME_DEPS = ["//:mainnet_nns_subnet_revision"]
+MAINNET_NNS_SUBNET_REVISION = mainnet_versions["subnets"]["tdb26-jop6k-aogll-7ltgs-eruif-6kk7m-qpktf-gdiqx-mxtrf-vb5e6-eqe"]
+MAINNET_APPLICATION_SUBNET_REVISION = mainnet_versions["subnets"]["io67a-2jmkw-zup3h-snbwi-g6a5n-rm5dn-b6png-lvdpl-nqnto-yih6l-gqe"]
+
+MAINNET_ENV = {
```

Comment:
I'm not too fond of lumping together env vars or deps like this and would prefer having them duplicated but up to you!